index.js

```diff
@@ -1 +1,20 @@
-// TODO: Start prototyping here
\ No newline at end of file
+import conf from './config.js';
+import express from 'express';
+
+class Server {
+    async run () {
+        this.app = express();
```

Comment:
The good practice is to implement the constructor and assign all class fields there. 
We do it to have one place to check all info that this class has.